src/app/store/store.js

```diff
@@ -69,27 +92,67 @@ export const resetFilters = () => async () => {
 
 export const searchResults = derived(
   [telemetrySearch, searchQuery], ([$telemetrySearch, $query]) => {
-    let candidates;
     let resultSet = [];
     if ($telemetrySearch.loaded) {
-      // , {limit:30}
-      candidates = $telemetrySearch.search($query).map((r, searchID) => ({ ...r, searchID }));
-      // const candidateIDs = candidates.map(c=>c.id);
-
-      // if same length and same names, then
-      // do not update resultSet.
-      // const sameLength = resultSet.length === candidates.length;
-      // const sameNames = candidates.every((c,i) => {
-      //     return resultSet[i] && resultSet[i].id && resultSet[i].id === c.id;
-      // })
-
-      // if (!sameLength || !sameNames) {
-      //     resultSet = candidates;
-      //     hovered = resultSet[0].searchID;
-      // }
-      resultSet = candidates;
+      resultSet = $telemetrySearch.search($query).map((r, searchID) => ({ ...r, searchID }));
     }
 
     return { results: resultSet, total: $telemetrySearch.length };
   },
 );
+
+// further derivations from the store
+
+export const hasDefaultControlFields = derived(store, ($store) => Object.values(CONFIG.fields)
+  .every((field) => field.values[0].key === $store[field.key]));
+
+// ///// probe querying infrastructure.
+
+// hook into store here.
+
+function getParamsForDataAPI(obj) {
+  return {
+    version: `${obj.version}`,
```

Comment:
Why is this one not like the others?

Reply:
I hit some issues with it not being a string before, but it may have actually been something else causing the problem, not this.